modules/extension/xsd/xsd-wfs/src/test/java/org/geotools/wfs/v2_0/WFSFeatureCollectionEncodingTest.java

```diff
@@ -328,4 +329,26 @@ Encoder encoderFormatting() {
         configuration.getProperties().add(org.geotools.gml2.GMLConfiguration.OPTIMIZED_ENCODING);
         return new Encoder(configuration);
     }
+
+    /** This test checks the Feature attributes are not pulled into the parent wfs:member */
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testEncodeFeatureMemberAttributes() throws Exception {
+        Map<Name, Object> attributesMap = new HashMap<>();
+        attributesMap.put(new NameImpl("example"), "123");
+
+        SimpleFeatureBuilder feature = new SimpleFeatureBuilder(store.getSchema("feature"));
+        feature.add("id1");
+        feature.add(new GeometryFactory().createPoint(new Coordinate(0, 0)));
+        feature.add(0);
+        feature.featureUserData(Attributes.class, attributesMap);
+
+        WFSConfiguration configuration = new WFSConfiguration();
+        Encoder e = new Encoder(configuration);
+        e.getNamespaces().declarePrefix("geotools", "http://geotools.org");
+        e.setIndenting(true);
+
+        Document doc = e.encodeAsDOM(feature.buildFeature("id1"), WFS.member);
+        Assert.assertFalse(doc.getDocumentElement().hasAttribute("example"));
```

Comment:
Not checking that the feature itself has the attribute, like in the other test?

Reply:
> Legit QA test failure:
> 
> ```
> 2023-03-07T04:19:03.9256796Z [INFO] PMD Failure: org.geotools.wfs.v2_0.WFSFeatureCollectionEncodingTest:48 Rule:wildcards Priority:3 No Wildcard Imports.
> ```
Holy wildcards Batman!  Thanks fixed!

Added feature attribute check.